docs/user-guides/federation.md

```diff
@@ -0,0 +1,154 @@
+    matchLabels:
+      monitoring.rhobs/stack: federate-cmo-ms
+
+  logLevel: info # 👈 debug for verbose logs
+  retention: 3h
+
+  prometheusConfig:
+    replicas: 2  # 👈 ensures that at least one prometheus is running during upgrade
+
+    enableRemoteWriteReceiver: true
+
+  alertmanagerConfig:
+    disabled: true
+
+  resources: # 👈 ensure that you provide sufficient amount of resources
+    requests:
+      cpu: 500m
+      memory: 1Gi
+```
+
+
+### Grant permission to Federate In-Cluster Prometheus
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: federate-cmo-ms-view
+  labels:
+    kubernetes.io/part-of: federate-cmo-ms
+    monitoring.rhobs/stack: federate-cmo-ms
+
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-monitoring-view
+subjects:
+- kind: ServiceAccount
+  name: federate-cmo-ms-prometheus  # 👈  ServiceAccount used in the prometheus deployed by ObO
+  namespace: federate-cmo
+```
+
+### Create ServiceMonitor for Federation
+
+```yaml
+apiVersion: monitoring.rhobs/v1
+kind: ServiceMonitor
+metadata:
+  name: federate-cmo-smon
+  namespace: federate-cmo
+  labels:
+    kubernetes.io/part-of: federate-cmo-ms
+    monitoring.rhobs/stack: federate-cmo-ms
+
+spec:
+  selector: # 👈 use the prometheus service to create a "dummy" target.
+    matchLabels:
+      app.kubernetes.io/managed-by: observability-operator
+      app.kubernetes.io/name: federate-cmo-ms-prometheus
+
+  endpoints:
+  - params:
+      'match[]': # 👈 scrape only required metrics from in-cluster prometheus
+        - '{__name__=~"container_cpu_.*", namespace="federate-cmo"}'
+        - '{__name__="container_memory_working_set_bytes", namespace="federate-cmo"}'
+        # NOTE: add additional metrics
+        # - '{__name__=~"container_cpu.*|container_memory_.*", namespace="kube-system"}'
+
+    # relabel example
+    relabelings:
+    - targetLabel: source
+      replacement: my-openshift-cluster
+    - action: replace
+      targetLabel: __address__
+      replacement: prometheus-k8s.openshift-monitoring.svc:9091
+    - action: labeldrop
+      regex: pod|namespace|service|endpoint|container
+
+    interval: 30s
+
+    # 👇 static part of the service-monitor
+    honorLabels: true
+
+    port: web
+    scheme: https
+    path: "/federate"
+
+    bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    bearerTokenSecret:
+      key: ""
+
+    tlsConfig:
+      ca:
+        configMap: # 👈 automatically created by serving-ca operator
+          key: service-ca.crt
+          name: openshift-service-ca.crt
+      serverName: prometheus-k8s.openshift-monitoring.svc
+
+```
+
+## Validation
+
+Verify if the setup works using Prometheus UI by port-forwarding the service
+created by ObO as follows
+
+```sh
+kubectl port-forward svc/federate-cmo-ms-prometheus 9090:9090 --address 0.0.0.0 -n federate-cmo
+
+# open: http://localhost:9090/targets
```

Comment:
we can probably come up with a `kubectl exec ... -- wget -O - http://localhost:9090/api/v2/targets | jq '...'` command that would provide the exact result.

Reply:
The intent is two fold here. I wanted to show that the UI is accessible for users to play around with. 
I can added the exec command as well.